config/src/ripgrep_config.rs

```diff
@@ -0,0 +1,173 @@
+// The code in this file has in large parts been copied from the ripgrep
```

Comment:
Don't we need a license specification somehow/somewhere?

Reply:
I've added a NOTICE file and tried my best to follow all rules about what should be in there..